src/sentry/tasks/derive_code_mappings.py

```diff
@@ -166,13 +166,17 @@ def log_project_codemappings(
     Log the code mappings that would be created for a project.
     """
     set_tag("project.slug", project.slug)
+    if code_mappings:
+        msg = f"Project {project.slug} would create these code mappings {code_mappings} based on {stacktrace_paths=}"
+        sentry_msg = "derive_code_mappings: NO code mappings would have been created."
+    else:
+        msg = f"Project {project.slug} would NOT create code mapping based on {stacktrace_paths=}"
```

Comment:
@armenzg wondering why we are putting this stuff in a message than doing something like `setTag('projectSlug', project.slug)`